articles/openshift/oshift-how-backup-migrate.md

```diff
@@ -0,0 +1,146 @@
+---
+title: How to backup, restore and migrate OpenShift resources | UKCloud Ltd
+description: Provides a starting point for using Velero to backup and restore cluster resources and migrate between clusters
+services: openshift
+author: Kieran O'Neill
+reviewer: 
+lastreviewed: 11/09/2019
+
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Backup, restore and migrate OpenShift resources
+toc_fullpath: How To/oshift-how-backup-migrate.md
+toc_mdlink: oshift-how-backup-migrate.md
+---
+
+# How to backup, restore and migrate OpenShift resources
+
+### Overview
+
+Velero is an open-source tool set that provides the ability to backup and restore kubernetes resources. The GitHub repository is avaialble at: <https://github.com/vmware-tanzu/velero>.
+
+This guide outlines how to:
+
+- Install Velero in your cluster (using our Elastic Cloud Storage product as the backup location)
+
+- Back up and restore namespaces to the same cluster
+
+- Restore a namespace to a new cluster
+
+Velero offers a tool called restic that enables backup of persistent volume data regardless of provider, however this guide doesn't cover restic as the tool is considered beta-quality. If you'd like to test it you can find more on restic here: <https://velero.io/docs/v1.1.0/restic/>.
+
+### Installing Velero
+
+1. Download Velero from <https://github.com/vmware-tanzu/velero/releases>.
+
+    You can deploy Velero onto OpenShift with the `velero install` command. To deploy Velero, you'll need compatible object storage: <https://velero.io/docs/v1.1.0/support-matrix/>. In this article we use UKCloud's Cloud Storage product (based on Dell EMC's ECS).
```

Comment:
```suggestion
    You can deploy Velero onto OpenShift with the `velero install` command. To deploy Velero, you'll need compatible object storage: <https://velero.io/docs/v1.1.0/support-matrix/>. In this article we use UKCloud's Cloud Storage product.
```
I'd remove the specifics about the service we use as it's not really relevant and could change in future.